behave_test_automation/pah_behave/features/page_objects/base_page.py

```diff
@@ -36,3 +40,17 @@ def wait_for_url(self, url) -> None:
         WebDriverWait(self.browser, 2).until(
             expected_conditions.url_to_be("http://localhost:8080/" + url)
         )
+
+    def page_has_loaded(self):
+        page_state = self.browser.execute_script(
+            'return document.readyState;'
+        )
+        return page_state == 'complete'
```

Comment:
```suggestion
        return self.browser.execute_script('return document.readyState') == 'complete'
```
That's a NitPick, don't mind that suggestion too much ;P

Reply:
No worries, changes applied